R/fftrees_wordstofftrees.R

```diff
@@ -44,42 +45,61 @@ fftrees_wordstofftrees <- function(x,
   )
 
 
-  # Clean up my.tree: ----
+  # Clean up and check my.tree: ------
 
-  # Split into one sentence:
+  # Collapse into one sentence:
   if (length(my.tree) > 1) {
     my.tree <- paste(my.tree, collapse = ". ")
   }
 
-  # Remove \n (can happen if my.tree has line breaks):
+
+  # Remove line breaks (\n):
   my.tree <- gsub(pattern = "\n", replacement = "", x = my.tree)
 
-  if (all(grepl(x$params$decision.labels[1], x = my.tree) == FALSE)) {
-    stop("Some decision.labels are not in my.tree.")
-  }
 
-  cue.names.l <- tolower(x$cue_names)
-  my.tree <- tolower(my.tree)
+  # Use lowercase spelling (for robustness against typos):
+  my.tree         <- tolower(my.tree)
```

Comment:
As above, I'm ok to leave this but I don't believe it is necessary.

Reply:
Here I would argue that creating 2 blocks is actually helpful (at least to me).  The code block 

```{r example}
  # Use lowercase spelling (for robustness against typos):
  my.tree         <- tolower(my.tree)
  cue.names.l     <- tolower(x$cue_names)
  decision.labels <- tolower(x$params$decision.labels)
```
 
seems much easier to parse than

```{r ex2}
  # Use lowercase spelling (for robustness against typos):
  my.tree <- tolower(my.tree)
  cue.names.l <- tolower(x$cue_names)
  decision.labels <- tolower(x$params$decision.labels)
```

And as the assignments are non-trivial (the 1st changes an object, the 2nd and 3rd use object info to create new objects), I find that the visual alignment can help understanding what's happening.  But as matters of taste vary, of course, I would not mind changing it back again either.

Reply:
Understood. Happy to incorporate these changes. 

There could be a world where there are several PRs from different authors with different styling preferences, which could lead to lots of review and inconsistency in the code base, but I'm happy to apply the Ostrich Algorithm https://en.wikipedia.org/wiki/Ostrich_algorithm for now.

Reply:
Thanks for the hint, hadn't see this before!  And I also will take a closer look at style guides, as I'm aware that my habits may be a bit idiosyncratic there.  On the other hand, I'm also sure that we'll manage, as long as we're aiming to be as clear as possible throughout all code & changes.  (And this **GitHub** thing helps a lot — am really fond of the transparent process.)